.clang-tidy

```diff
@@ -0,0 +1,24 @@
+Checks: '
+  *,
+  -cert-err58-cpp,
+  -cppcoreguidelines-pro-type-union-access,
+  -fuchsia-default-arguments,
+  -fuchsia-overloaded-operator,
+  -fuchsia-statically-constructed-objects,
+  -google-readability-todo,
+  -hicpp-signed-bitwise
+  '
```

Comment:
Could you explain why these are to be excluded?

Reply:
Arbitralny wybór tego co się często pojawia w tym kodzie i czego na pewno nie będziemy uznawać za błąd / poprawiać.